include/deal.II/grid/construction_util.h

```diff
@@ -0,0 +1,120 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_grid_construction_util_h
+#define dealii_grid_construction_util_h
```

Comment:
I would not shorten the file name since it is not mnemonic. I would either use `utilities` or `construction_utilities`, with a slight preference for the second.

Reply:
It is true that we shorten `triangulation` in `tria` consistently, however this is only true for triangulation (and already troublesome, IMHO). Introduring other short names around makes life very difficult... we have `base/utilities`, it would make sense to have `grid/utilities`, or not to add anything else, and just add it to `GridTools`. For this, though, I'll let others comment.

Reply:
I have updated the code. I would prefer if we could keep a separate file for these functionality.